package.json

```diff
@@ -28,22 +28,22 @@
   },
   "homepage": "https://github.com/RealOrangeOne/react-native-mock#readme",
   "devDependencies": {
-    "babel": "=6.5.2",
-    "babel-cli": "=6.9.0",
-    "babel-core": "=6.9.0",
-    "babel-eslint": "=6.0.4",
-    "babel-preset-airbnb": "=2.0.0",
-    "babel-preset-react-native": "=1.8.0",
-    "chai": "=3.5.0",
+    "babel": "6.5.2",
+    "babel-cli": "6.9.0",
+    "babel-core": "6.9.0",
+    "babel-eslint": "6.0.4",
+    "babel-preset-airbnb": "2.0.0",
+    "babel-preset-react-native": "1.8.0",
+    "chai": "3.5.0",
     "eslint": "2.10.2",
-    "eslint-config-airbnb": "=9.0.1",
-    "eslint-plugin-import": "=1.8.0",
-    "eslint-plugin-jsx-a11y": "=1.2.2",
-    "eslint-plugin-react": "=5.1.1",
-    "eslint-plugin-react-native": "=1.0.2",
-    "mocha": "=2.5.3",
+    "eslint-config-airbnb": "9.0.1",
+    "eslint-plugin-import": "1.8.0",
+    "eslint-plugin-jsx-a11y": "1.2.2",
+    "eslint-plugin-react": "5.1.1",
+    "eslint-plugin-react-native": "1.0.2",
+    "mocha": "2.5.3",
     "react": "^15.4.0",
-    "react-native": ">=0.27.0"
+    "react-native": "0.38.0"
```

Comment:
Can you leave this as a loose dependancy, so it upgrades automatically. Will help with tests